dev/cognitiveservices/data-plane/Language/generate-questionanswers.json

```diff
@@ -0,0 +1,643 @@
+      },
+      "enum": [
+        "InvalidRequest",
+        "InvalidArgument",
+        "Unauthorized",
+        "Forbidden",
+        "NotFound",
+        "TooManyRequests",
+        "InternalServerError",
+        "ServiceUnavailable"
+      ]
+    },
+    "InnerErrorCode": {
+      "type": "string",
+      "description": "Human-readable error code.",
+      "x-ms-enum": {
+        "name": "InnerErrorCode",
+        "modelAsString": true
+      },
+      "enum": [
+        "InvalidRequest",
+        "InvalidParameterValue"
+      ]
+    },
+    "Language": {
+      "type": "string",
+      "description": "Language of the document content. This is BCP-47 representation of a language. For example, use \"en\" for English; \"es\" for Spanish etc. If not set, use \"en\" for English as default."
+    },
+    "StringIndexType": {
+      "type": "string",
+      "description": "Specifies the method used to interpret string offsets.  Defaults to Text Elements (Graphemes) according to Unicode v8.0.0. For additional information see https://aka.ms/text-analytics-offsets.",
+      "default": "TextElements_v8",
+      "enum": [
+        "TextElements_v8",
+        "UnicodeCodePoint",
+        "Utf16CodeUnit"
+      ],
+      "x-ms-enum": {
+        "name": "StringIndexType",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "TextElements_v8",
+            "description": "Returned offset and length values will correspond to TextElements (Graphemes and Grapheme clusters) confirming to the Unicode 8.0.0 standard. Use this option if your application is written in .Net Framework or .Net Core and you will be using StringInfo."
+          },
+          {
+            "value": "UnicodeCodePoint",
+            "description": "Returned offset and length values will correspond to Unicode code points. Use this option if your application is written in a language that support Unicode, for example Python."
+          },
+          {
+            "value": "Utf16CodeUnit",
+            "description": "Returned offset and length values will correspond to UTF-16 code units. Use this option if your application is written in a language that support Unicode, for example Java, JavaScript."
+          }
+        ]
+      }
+    },
+    "QnAGenerationRequestBody": {
+      "type": "object",
+      "description": "QnA Generation job with documents and qnagenparameters.",
+      "additionalProperties": false,
+      "properties": {
+        "documents": {
+          "type": "array",
+          "description": "The set of documents to process as part of qna generation job.",
+          "items": {
+            "$ref": "#/definitions/QnAGenDocument"
+          }
+        },
+        "qnaGenerationParameters": {
+          "description": "Parameters that define max number of QnAs to be generated, fetch raw documents or not, etc.",
+          "$ref": "#/definitions/QnAGenParameters"
+        }
+      }
+    },
+    "QnAGenerationResult": {
```

Comment:
I recommend calling this a `GenerateResult` to match the pattern that other endpoints have used in Language.

Reply:
called it GenerationResult